presto-main/src/main/java/com/facebook/presto/server/PrestoSystemRequirements.java

```diff
@@ -75,6 +76,14 @@ private static void verifySlice()
         }
     }
 
+    public static void verifySystemTimeIsCurrent()
+    {
+        int currentYear = DateTime.now().year().get();
+        if (currentYear < 2015) {
```

Comment:
If I understand properly from @dain's issue, this is more about having a roughly correct year (e.g. not something totally weird, like 1900 or something) than having the exact current year. I can change the name of the method to reflect that though.